### DISCUSSION TITLE
Misleading outline symbol for R variables

### DISCUSSION THREAD

**Discussion by kv9898** [posit-dev/positron#5065] [Feedback] [answered]
Currently, the outline symbol for variables looks like this:
code:
```r
a <- 1
```
outline:
![image](https://github.com/user-attachments/assets/315eb429-03f6-4bf0-9046-46c8aec00437)
This kind of hints at a "function" to me.

I wonder a `VARIABLE`  symbol rather than the current `OUTLINE` symbol would be more appropriate. An example of `VARIABLE` symbols would be:
![image](https://github.com/user-attachments/assets/c8cf6a5d-e8be-4757-bc68-36d90eef70f6)

**Comment by testlabauto** [comment, marked as answer]
Hi.  I think you make a great point.  Thanks for pointing this out.  I have created an issue based on your input:
https://github.com/posit-dev/positron/issues/5071
